docs/content/documentation.json

```diff
@@ -2506,7 +2506,7 @@
             },
```

Comment:
This doc should not exist. There is a gitignore rule for it but somehow it keeps popping up...